qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -631,11 +631,12 @@ def _on_history_trigger(self):
 
     @pyqtSlot(QUrl, 'QAuthenticator*')
     def _on_authentication_required(self, url, authenticator):
-        # FIXME:qtwebengine support .netrc
-        answer = shared.authentication_required(
-            url, authenticator, abort_on=[self.shutting_down,
-                                          self.load_started])
-        if answer is None:
+        netrc = shared.netrc_authentication(url, authenticator)
+        if not netrc:
```

Comment:
Wouldn't it be easier to call `shared.netrc_authentication` from `shared.authentication_required` and handling it in there entirely?